### PR TITLE
fix(global): rename default status color to custom

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/5175

Also makes a status variation required for the alert component. Previously the alert with no variant would give you the "default", aka "custom" status styling. Now the alert without a variation is unstyled and you have to add `.pf-m-custom`.

### CODE REVIEW

File: src/patternfly/components/Alert/alert.scss [hunk 1]
```diff
@@ -119,6 +124,13 @@
   border-top: var(--pf-c-alert--BorderTopWidth) solid var(--pf-c-alert--BorderTopColor);
   box-shadow: var(--pf-c-alert--BoxShadow);
 
+  &.pf-m-custom {
+    --pf-c-alert--BorderTopColor: var(--pf-c-alert--m-custom--BorderTopColor);
+    --pf-c-alert__icon--Color: var(--pf-c-alert--m-custom__icon--Color);
+    --pf-c-alert__title--Color: var(--pf-c-alert--m-custom__title--Color);
+    --pf-c-alert--m-inline--BackgroundColor: var(--pf-c-alert--m-inline--m-custom--BackgroundColor);
```
Comment (thatblindgeye) [inline src/patternfly/components/Alert/alert.scss:131]: Doesn't look like `--pf-c-alert--m-inline--m-custom--BackgroundColor` is defined anywhere

File: src/patternfly/components/Popover/popover-title-text.hbs [hunk 2]
```diff
@@ -4,7 +4,7 @@
   {{/if}}>
     {{#if popover--IsAlert}}
       {{#> screen-reader}}
-      {{#if popover--IsDefaultAlert}}
+      {{#if popover--IsCustomAlert}}
         Default
```
Comment (thatblindgeye) [inline src/patternfly/components/Popover/popover-title-text.hbs:8]: Similar to above, would we want this text to be "Custom" instead?

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

**Review by thatblindgeye — COMMENTED** [review]
Looks good! Only other thing is the screenshot for the "custom alert popover" example needs to be updated.

**Comment by mcoker** [comment]
@thatblindgeye great catches! The full-page example screenshots are updated prior to a release, usually with the versioned commit that adds the release notes. Though you can run the job and push to this PR if you'd like! The instructions are at https://github.com/patternfly/patternfly/#update-screenshots Note that it may update more screenshots than this one since that job isn't run with each change and there have been a lot of changes since the last run.

File: src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md
Comment (srambach) [inline src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md:1]: Line 127 - Should add '.pf-m-custom' as an acceptable modifier (or just remove the list and simply say always to use it with a state modifier)

**Review by srambach — APPROVED** [review]
:+1 LGTM

**Pull request merged**